src/clients/python/experimental_api_v2/library/httpclient.py

```diff
@@ -780,9 +780,9 @@ def unregister_cuda_shared_memory(self,
 
     def infer(self,
               inputs,
+              outputs,
```

Comment:
'outputs' is an optional field. According to specification if the 'outputs' are not provided then results for all output tensor of the model should be returned in default setting.

Reply:
Will make these changes. Currently the server returns only outputs for the outputs mentioned in the request header

Reply:
Adding a ticket to track this